CONTRIBUTING.md

```diff
@@ -1,9 +1,28 @@
-# How to contribute
 
+This is a copy of Lukas Eder's JOOq Contributing.md guide - I'll be modiying it for this repository shortly.
+Credit for original goes to Lukas Eder's Jooq CONTRIBUTING.md file.  Original can be found at: https://raw.githubusercontent.com/jOOQ/jOOQ/master/CONTRIBUTING.md
 
-* The integration tests need a running Docker infrastructure.
```

Comment:
I'm not sure of the need to drastically change the contribution guide - would you mind elaborating on why?

Reply:
Great! Are you back? 
My apologies, that should not have made it into this PR.  You can close this out as the main content is a duplicate of 242 anyway.

Reply:
Yes, sorry about that. Ok, will close this PR.